<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopicSelector</class>
 <widget class="QWidget" name="TopicSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Select topics to follow</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2_saveTopics">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Save Current Topic List</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_loadTopics">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Load Topic List</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QListView" name="TopicListAll">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="allSelect_allTopics">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="noneSelect_allTopics">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_filter_allTopics">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="pushButton_selectTopics">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_deselectTopics">
           <property name="minimumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListView" name="TopicListSelected">
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="allSelect_selectedTopics">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="font">
              <font>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>All</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="noneSelect_selectedTopics">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>None</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_filter_selectedTopics">
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
